docs/modules/ROOT/pages/relay.adoc

```diff
@@ -98,6 +98,34 @@ In addition to sending transactions, the Relayer can also sign arbitrary message
 const signResponse = await relayer.sign({ message });
 ----
 
+[[relayer-info]]
+=== Relayer Info
+
+A relayer's address can be retrieved using the `getAddress` method of the `DefenderRelaySigner` class.
+
+[source,jsx]
+----
+const addr = await relayer.getAddress();
```

Comment:
Ah, I understand now what you meant. Still, in the examples right before the one you added, `relayer` refers to a `Relayer` object, not to a `DefenderRelaySigner`. I'd rename this one to `signer` for clarity.

```suggestion
const addr = await signer.getAddress();
```

Reply:
@ylv-io bump!